wandb/sdk/wandb_artifacts.py

```diff
@@ -1024,6 +1042,9 @@ def _prepare_fn() -> "internal_api.CreateArtifactFileSpecInput":
                             for header in (resp.upload_headers or {})
                         },
                     )
+                if self._disable_cache == True:
+                    os.remove(entry.local_path)
```

Comment:
Can we be guaranteed that entry.local_path is always pointing to our copy? It is possible, but i have a gut feeling like this could be pointing to a user file on the local machine